### PR TITLE
Mainline aries cloudagent

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
Switch to a publically published version of `aries-cloudagent-python` rather than our fork.
Screenshot 2022-03-14 at 16 04 58

### CODE REVIEW

File: docker/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+version: '3'
+
+services:
+  # cloudagent-db:
```
Comment (n3op2) [inline docker/docker-compose.yaml]: I guess this is for later, but what is the story behind this db?

Comment (mattdean-digicatapult) [reply docker/docker-compose.yaml]: Ah well spotted. I should probably clean that up. Basically the cloud-agent "should" be able to use a postgres backend which would be preferable when we look at mapping this to the cloud. For now though I'll clean this up and just use the local wallet

Comment (mattdean-digicatapult) [reply docker/docker-compose.yaml]: Removed

File: docker/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      dockerfile: Dockerfile.init
+    container_name: veritable-demo-init
+    networks:
+      - von
+  cloudagent-issuer:
+    image: bcgovimages/aries-cloudagent:py36-1.16-1_0.7.3
+    container_name: veritable-cloudagent-issuer
+    command: 'start --inbound-transport http 0.0.0.0 8000 --inbound-transport ws 0.0.0.0 8001 --admin 0.0.0.0 8002'
+    restart: on-failure
+    environment:
```
Comment (n3op2) [inline docker/docker-compose.yaml:16]: that is a lot of variables, maybe abstract into a file? or have something like default config for some like

```
      - ACAPY_AUTO_PROVISION=true
      - ACAPY_WALLET_TYPE=indy
      ...
      - ACAPY_AUTO_ACCEPT_INVITES=true
      - ACAPY_AUTO_ACCEPT_REQUESTS=true
      - ACAPY_AUTO_STORE_CREDENTIAL=true
      - ACAPY_PRESERVE_EXCHANGE_RECORDS=true
```
Looks like they are being shared across all personas

**Review by n3op2 — APPROVED** [review]

File: docker/docker-compose.yaml (hunk 2 above)
Comment (mattdean-digicatapult) [reply docker/docker-compose.yaml:16]: not a bad idea, I can do that. There are some differences between personas but I'll pull out the common stuff

**Review by jonmattgray — APPROVED** [review]

File: docker/docker-compose.yaml (hunk 2 above)
Comment (mattdean-digicatapult) [reply docker/docker-compose.yaml:16]: I will do this in a separate PR to keep moving

**Pull request merged**